filesystem/iso9660/finalize.go

```diff
@@ -713,6 +713,8 @@ func (fs *FileSystem) Finalize(options FinalizeOptions) error {
 	b = terminator.toBytes()
 	f.WriteAt(b, int64(location)*int64(blocksize))
 
+	os.RemoveAll(fs.workspace)
```

Comment:
Should we report an error here even if the iso was created successfully?

Reply:
Maybe just a warning. Even if we didn't, we should capture the potential error so linters don't complain about it, as `_ = os.RemoveAll(fs.workspace)`

But I think really you are right. Just capture it and warn. At least let the user know that something is lingering.

Reply:
> Just capture it and warn

This would be great, but I'm not sure how to accomplish it. We don't have a logger and there's not really a distinction between error types returned typically.
What did you have in mind for this?

Reply:
Oh good point. I had forgotten about that. These libraries are not meant to be interacting directly that way. The right thing to do probably is to have various kinds of custom errors, pass those up, and let the caller know that the final state was successful, but that there were secondary errors.

That is far more than I want to think about in this PR, so let's just take your input to date.